### PR TITLE
Perfgrapher

### PR DESCRIPTION
**Pull request by jagged-teeth**
Perfgrapher init PR,  parsing with dummy values and dummy observer class.

### CODE REVIEW

File: Source/Private/PerfGrapherCommandlet.cpp [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    UE_LOG( LogPerfGrapher, Display, TEXT( "Raw params: %s" ), *params );
+    UE_LOG( LogPerfGrapher, Display, TEXT( "Found %d tokens, %d switches, %d params" ), tokens.Num(), switches.Num(), params_map.Num() );
+
+    for ( const auto & param : params_map )
+    {
+        UE_LOG( LogPerfGrapher, Display, TEXT( "In Param: %s = %s" ), *param.Key, *param.Value );
+    }
+
+    // Set defaults before parsing overrides
+    out_params.MapName = TEXT( "L_OW" );
```
Comment (TheEmidee) [inline Source/Private/PerfGrapherCommandlet.cpp]: no project specific map in this plugin

**Review by TheEmidee — CHANGES_REQUESTED** [review]

File: Source/Private/PerfGrapherCommandlet.cpp (hunk 1 above)
Comment (jagged-teeth) [reply Source/Private/PerfGrapherCommandlet.cpp]: Yeah I know, I wanted to keep this for now to allow for more practical testing. I will remove it.

Comment (jagged-teeth) [reply Source/Private/PerfGrapherCommandlet.cpp]: replaced it with a note and default_map. To replace with our map without pushing it.

Comment (TheEmidee) [reply Source/Private/PerfGrapherCommandlet.cpp]: leave it to empty

Comment (jagged-teeth) [reply Source/Private/PerfGrapherCommandlet.cpp]: please merge, will do in future PR

Comment (jagged-teeth) [reply Source/Private/PerfGrapherCommandlet.cpp]: Should be fine.

**Review by TheEmidee — APPROVED** [review]

**Pull request merged**